src/components/Structure/ViewerAndEntries/index.tsx

```diff
@@ -291,16 +291,29 @@ class StructureView extends PureComponent<Props, State> {
     return hits;
   }
 
+  getEntryNames() {
+    const accessionToName: Record<string, NameObject | string> = {};
+    if (this.props.matches) {
+      for (const match of this.props.matches) {
+        const entryName = match.metadata.name;
+        const entryAccession = match.metadata.accession;
+        accessionToName[entryAccession] = entryName || entryAccession;
+      }
+    }
+    return accessionToName;
+  }
+
   createEntryMap() {
     const memberDBMap: Record<
       string,
       Record<string, Record<string, Array<EntryHit>>>
-    > = { pdb: {} };
+    > = {};
 
     if (this.props.matches) {
       // create matches in structure hierarchy
+
       for (const match of this.props.matches) {
-        const entry = match.metadata.accession;
+        let entry = match.metadata.accession;
```

Comment:
We're not reassigning a value to `entry`, so leave it a `const`

```suggestion
        const entry = match.metadata.accession;
```